### PR TITLE
Default Career and Education labels for events

### PR DESCRIPTION
**Pull request by maarten0912**
**Please describe what your PR is fixing**
Default the labels "Career" and "Education" to company and educational events respectively.

**Concretely, which issues does your PR solve? (Please reference them by typing `Fixes/References Inter-Actief/amelie#<issue_id>`)**
Fixes Inter-Actief/amelie#678

**Does your PR change how we process personal data, impact our [privacy document](https://www.inter-actief.utwente.nl/privacy/), or modify (one of) our data export(s)?**
yes

**Does your PR include any django migrations?**
yes

**Does your PR include the proper translations (did you add translations for new/modified strings)?**
no, my PR does not include translations

**Does your PR include CSS changes (and did you run the `compile_css.sh` script in the `scripts` directory to regenerate the `compiled.css` file)?**
no, my PR does not include CSS changes

**Does your PR need external actions by for example the System Administrators? (Think about new pip packages, new (local) settings, a new regular task or cronjob, new management commands, etc.)?**
no

**Did you properly test your PR before submitting it?**
yes

### CODE REVIEW

File: amelie/education/migrations/0005_add_default_education_label.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('education', '0004_auto_20221028_1502'),
+    ]
+
+    operations = [
+        migrations.RunPython(create_default_education_label),
+    ]
```
Comment (WKobes) [inline amelie/education/migrations/0005_add_default_education_label.py:26]: newline

**Comment by WKobes** [comment]
@maarten0912 Good work on this. I was just thinking, do we need to add these labels to the database? Since we know it are either company or educational events, couldn't we just insert these labels at the front end for these events?

It would save some database space and only slightly complicate template code. What do you think?

**Comment by maarten0912** [comment]
@WKobes I surely think it’s possible and could work well, but in (our) practice the ‘Education’ and ‘Career’ labels exist already in the database. I cannot think of any case where we would not want these labels to exist for “normal” activities, but if there is, I can imagine that other way of solving it. Maybe this also shows some flaws in the way that events are currently implemented (or used); for example having both education events but also activities with the education label.

**Comment by maarten0912** [comment]
Finished testing the migrations, fixed issue and added small comment!

**Review by Kurocon — APPROVED** [review]

**Pull request merged**